src/dso_api/dynamic_api/views/doc.py

```diff
@@ -12,11 +12,10 @@ class DocsOverview(TemplateView):
     template_name = "dso_api/dynamic_api/docs/overview.html"
 
     def get_context_data(self, **kwargs):
-        datasets = (ds.schema for ds in Dataset.objects.api_enabled().db_enabled().all())
+        datasets = (ds for ds in Dataset.objects.api_enabled().db_enabled().all())
```

Comment:
` (ds for ds in Dataset.objects.api_enabled().db_enabled().all())`
mag natuurlijk geschreven worden als `Dataset.objects.api_enabled().db_enabled().all()`

Reply:
Goeie. Die gaat mee in de volgende ronde.